tensorflow/lite/micro/testing/size_xtensa_binary.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Measures the size of a Qualcomm Hexagon binary by parsing the output of
+# xt-size. If an optional list of symbols is provided, the symbols' sizes
+# are excluded from the total. This is useful when the binary contains symbols
+# that are only used during testing.
+#
+# First argument is the binary location.
```

Comment:
This Script seems to be very similar with size_hexagon_binary.sh.   WDYT of merging these two scripts instead of having them separately?